test_check_if_anagrams.py

```diff
@@ -1,7 +1,24 @@
 def are_anagrams(str1, str2):
-  # Remove the `return None` code below and add your own code.
-  return None
 
+  i = 0
+  j = 0
```

Comment:
I'd like to encourage you to name your variables more descriptively, even if they are "just" counting variables. This will be helpful to others reading your code and to you, if you return to your code after some time.